Try(
	Show(a);
, // Catch
	show(exception_msg);
);
